radio/src/gui/gui_common.cpp

```diff
@@ -832,7 +832,13 @@ bool isTrainerModeAvailable(int mode)
 bool modelHasNotes()
 {
   char filename[sizeof(MODELS_PATH)+1+sizeof(g_model.header.name)+sizeof(TEXT_EXT)] = MODELS_PATH "/";
-  char *buf = strcat_currentmodelname(&filename[sizeof(MODELS_PATH)]);
+  char *buf = strcat_currentmodelname(&filename[sizeof(MODELS_PATH)], 0);
+  strcpy(buf, TEXT_EXT);
+  if (isFileAvailable(filename)) {
+    return true;
+  }
+
+  buf = strcat_currentmodelname(&filename[sizeof(MODELS_PATH)], ' ');
```

Comment:
Both this and the last test are returning `MODEL 1` instead of one of them returning `MODEL_1`, so never check the alternate filename.

Reply:
Yes, but model 1.bin is not a valid file name for model, isn't it?
So the space treatment was added to "model name" - not "modelFileName.bin".
Not too difficult to do, but do we need it?

Reply:
My bad, so MODEL1 is the name.
Now I'm lost: you said that TX16S was working as expected.
This code is common or all platforms.
Before line 842 was added, this function did not recognize MODEL_1.txt as a valid notes file for MODEL 1.

Reply:
My bad, so MODEL1 is the name.
Now I'm lost: you said that TX16S was working as expected.
This code is common or all platforms.
Before line 841 was added, this function did not recognize MODEL_1.txt as a valid notes file for MODEL 1.